src/resources/LibraryStyles.css

```diff
@@ -154,13 +153,18 @@ input {
 }
 
 .Arrow {
-    padding: 3px 6px 6px 12px;
-    height: 11px;
-    width: 11px;
+    height: 35px;
+    width: 35px;
+}
+
+.BodyIndentationWithLine,
+.BodyIndentationWithoutLine {
+    margin-bottom: 18px;
+    padding-left: 18px;
 }
 
-.BodyIndentation {
-    padding: 8px;
+.BodyIndentationWithLine {
```

Comment:
Hi @fanwgwg,

Newly introduced elements such as `.BodyIndentationWithLine` and `.BodyIndentationWithoutLine` are actually just margins and do not need to be separate elements.

We can assign multiple class names in React:
```ts
className={LibraryItemBodyContainer}                     /* BodyIndentationWithoutLine */
className={LibraryItemBodyContainer HasExpanderLine}        /* BodyIndentationWithLine */
```

```css
.HasExpanderLine {
    margin-left: 18px;
}

.HasExpanderLine::before {
    content: "";
    border-right: 1px solid #808080;
    margin-bottom: 18px;
}
```